dojo/settings/settings.dist.py

```diff
@@ -625,6 +633,10 @@ def generate_url(scheme, double_slashes, user, password, host, port, path):
     'update-findings-from-source-issues': {
         'task': 'dojo.tasks.async_update_findings_from_source_issues',
         'schedule': timedelta(hours=3),
+    },
+    'compute-sla-age-and-notify': {
+        'task': 'dojo.tasks.async_sla_compute_and_notify',
+        'schedule': crontab(hour=7, minute=30),
```

Comment:
Why 7.5 hours and not once per day?

Reply:
Unless I am mistaken, it is every day at 7:30AM (TZ being defined by Django).

Reply:
https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#crontab-schedules